30-queues_stacks.cs

```diff
@@ -0,0 +1,42 @@
+string? txt = Console.ReadLine();
+
+Solution obj = new Solution();
+
+foreach (char cr in txt)
+{
+	obj.PushCharacter(cr);
+	obj.EnqueueCharacter(cr);
+}
+
+bool isPalindrome = true;
+
+for (int i = 0; i < txt.Length / 2; i++)
+{
+	if (obj.PopCharacter() != obj.DequeueCharacter())
+	{
+		isPalindrome = false;
+
+		break;
+	}
+}
+
+
+if (isPalindrome)
+{
+	Console.Write($"The word, {txt}, is a palindrome.");
+}
+else
+{
+	Console.Write($"The word, {txt}, is not a palindrome.");
+}
```

Comment:
Here I see code repetition and think about a better solution to avoid repetition.